### PR TITLE
Translations & Keybind fix

### PR DESCRIPTION
**Pull request by OsiJr**
@asunawu confirmed but @jproust has not

Fixes [AB#2233](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/2233)

### CODE REVIEW

**Review by samsinsane — DISMISSED** [review]

**Comment by samsinsane** [comment]
Wait for @jproust to review before merging.

**Review by jproust — DISMISSED** [review]

**Review by samsinsane — APPROVED** [review]

**Pull request merged**
